staging/src/k8s.io/client-go/tools/cache/reflector.go

```diff
@@ -732,6 +732,8 @@ func watchHandler(start time.Time,
 	stopCh <-chan struct{},
 ) error {
 	eventCount := 0
+	initialEventsEndBookmarkWarningTicker := newInitialEventsEndBookmarkTicker(name, clock, start, exitOnInitialEventsEndBookmark)
```

Comment:
you effectively only use the information whether `exitOnInitialEventsEndBookmark` is non-nil

instead of passing the pointer there and risking that someone in the future will touch it, can we just pass `eexitOnInitialEventsEndBookmark != nil`

Reply:
since the pointer wasn't stored there was no risk that someone could use it. anyway i've adapted the code to your suggestion.